akka-ssl-config/src/main/resources/reference.conf

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+
+# ssl configuration
+ssl-config {
+
+  logger = "com.typesafe.sslconfig.akka.util.AkkaLoggerBridge"
```

Comment:
but it's bad practice to try to override someone else's reference.conf in a reference.conf, classpath ordering matters.

To me it looks like the intention was to use `akka.ssl-config.logger` here